.github/workflows/update-session-issues.yml

```diff
@@ -11,13 +11,16 @@ on:
     - cron: '0 1,2 * * 4' # Asia/Oceania
 
 jobs:
-  update-design-review-issues:
+  update-session-issues:
     if: github.repository == 'ampproject/amphtml'
-    name: Update Design Review Issues
+    name: Update Session Issues
     runs-on: ubuntu-latest
     steps:
-      - name: Update Design Review Issues
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
```

Comment:
nit: For better readability and to follow the existing convention across other yaml config files, remove blank lines between steps, and only retain the ones between top-level sections (`name`, `on`, `jobs`).